system_metrics_collector/README.md

```diff
@@ -29,20 +52,25 @@ Parameter name: publish_period
 
 ## Usage:
 
-You can run the following command to generate CPU and Memory statistics. for both, all the system and the node running:
+### ROS2 Launch
+The system CPU and system memory can be launched by using [ros2 launch](https://github.com/ros2/launch):
+```
+ros2 launch system_metrics_collector system_cpu_and_memory.launch.py
+```
 
+### Manual Execution
+You can run the example main (todo rename as example) which will manually start the system CPU, system memory, process
+CPU and process memory measurement nodes.
 ```
 ros2 run system_metrics_collector main
 ```
 
 Change `publish_period` or `measurement_period` using `--ros-args`:
-
 ```
```

Comment:
```suggestion
 ```sh
```

Reply:
This suggestion applies to a few sections of the `README.md`

Reply:
Done here and applied to the rest of the README